meta-luneos/recipes-core/packagegroups/packagegroup-luneos-extended.bb

```diff
@@ -176,6 +176,7 @@ QEMU_RDEPENDS = " \
 
 RDEPENDS:${PN}:append:qemux86 = " ${QEMU_RDEPENDS}"
 #RDEPENDS:${PN}:append:qemux86-64 = " ${QEMU_RDEPENDS} waydroid"
+RDEPENDS:${PN}:append:qemux86-64 = " ${QEMU_RDEPENDS}"
```

Comment:
Maybe remove the commented version above now?

Reply:
Done